config.default.yaml

```diff
@@ -24,11 +24,16 @@ scenario:
   # B for biomass supply, I for industry, shipping and aviation
   # solarx or onwindx changes the available installable potential by factor x
   # dist{n} includes distribution grids with investment cost of n times cost in data/costs.csv
+  # solar+c0.5 reduces the capital cost of solar to 50\% of reference value
```

Comment:
Is the intention that when you do solar+p0.5 it reduces the potential p_nom_max by 50%? This is like solar0p5, right?

Reply:
We could replace solarx and onwindx for altering the installable potentials with your notation.

Reply:
Done here https://github.com/PyPSA/pypsa-eur-sec/pull/90/commits/fd6239db5c6e5f0a711f8fa99e9048b0d023f3b4